internal/executor/build/task.go

```diff
@@ -89,11 +89,11 @@ func (task *Task) ProtoCommands() []*api.Command {
 	return result
 }
 
-func (task *Task) UniqueName() string {
```

Comment:
Maybe we should simply keep this as is for now, and change later this once we figure out the `UniqueName()` implementation?

Reply:
I've added smart capitalization of `task`. PTAL 75ed0691276f2fa76f94fe6f31fc0d72c65b5a15